blocks/quilted-image-block/features/quilted-image/default.jsx

```diff
@@ -0,0 +1,230 @@
+import React from "react";
+import PropTypes from "@arc-fusion/prop-types";
+import { useContent } from "fusion:content";
```

Comment:
@ecarlisle were you talking about having trouble with the linter? let me know if you want help with this as you shouldn't need to do `git push --no-verify`

Reply:
@JackHowa : Yeah, sorry, going to clean this up. 👍🏼